### DISCUSSION TITLE
个人建议

### DISCUSSION THREAD

**Discussion by orangbus** [imsyy/SNav#22] [Ideas]
可以在搜索栏下面添加6-8个常用的地址快捷方式，类似google的主页一样的，点击更多的时候背景虚化，弹出所有的快捷地址。
第一次遇到一个设计风格跟我想要的是一致的，加油

**Comment by imsyy** [comment]
收到，近期会开始重构

**Comment by xiaomailang** [comment]
真的重构吗，太棒了！上面的建议也很不错，添加几个常用高频的地址快捷方式，但建议加个开关，有需要的开，不需要的关，不影响整个主页简洁。功能上能不能实现一个备忘录功能

**Comment by imsyy** [comment]
> 真的重构吗，太棒了！上面的建议也很不错，添加几个常用高频的地址快捷方式，但建议加个开关，有需要的开，不需要的关，不影响整个主页简洁。功能上能不能实现一个备忘录功能

ok 👌

**Comment by Kirara-Tianqing** [comment]
原来的感觉挺好的呀，隐藏式开关，界面非常简洁，我用双屏没事的话一般一个屏幕玩游戏一个挂着imsyy的导航页，简洁的页面太赏心悦目了

**Reply by Kirara-Tianqing** [reply]
对了，大佬能把时间的秒数显示加上吗？有时候处理一些东西需要看秒数，老看系统桌面的时间不方便

**Reply by imsyy** [reply]
嗯，主页面会保持基本不变的，显示秒数到时候会在设置里面有的

**Reply by Kirara-Tianqing** [reply]
hhh，期待大佬重构完成
